javascript/javascript3/week3/circle.js

```diff
@@ -0,0 +1,47 @@
+const canvas = document.getElementById("canvas");
+let context = canvas.getContext("2d");
+let canvasHeight = window.innerHeight;
+let canvasWidth = window.innerWidth;
+canvas.height = canvasHeight;
+canvas.width = canvasWidth;
+
+canvas.style.background = "#ff8";
+
+class Circle {
+  constructor(xpos, ypos, radius, color) {
+    this.xpos = xpos;
+    this.ypos = ypos;
+    this.radius = radius;
+    this.color = color;
+  }
+  draw(context) {
+    context.beginPath();
+    context.arc(this.xpos, this.ypos, this.radius, 0, 2 * Math.PI, false);
+    context.fillStyle = circleColor;
```

Comment:
I would create a method doing this and setting linewidth. Imho that reads better, and it also mean you can reuse it should you need it.

Reply:
Thanks for suggestion I will change it.